pkg/cli/logout/logout.go

```diff
@@ -149,3 +154,25 @@ func deleteTokenFromConfig(config kclientcmdapi.Config, pathOptions *kclientcmd.
 	}
 	return kclientcmd.ModifyConfig(pathOptions, config, true)
 }
+
+// TokenToObjectName returns the oauthaccesstokens object name for the given raw token,
+// i.e. the sha256 hash prefixed with "sha256~".
+func TokenToObjectName(code string) string {
```

Comment:
nit: lower-case these new funcs as they are not used externally. Otherwise :+1:

Reply:
Similarly the other two functions lower-case as well.